### PR TITLE
update

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**
fix failure in
- onboarding
- MT5 and DerivX

### CODE REVIEW

File: cypress/e2e/wipWallet/derivx.cy.js [hunk 1]
```diff
@@ -56,12 +56,17 @@ describe('WALL-3252 - Add derivx account', () => {
...
+    const Text = Cypress.$(
+      ":contains('This account offers CFDs on a highly customisable CFD trading platform.')"
+    )
+    if (Text.length > 0) {
+      clickAddDerivxButton()
+      verifyDerixCreation('Real')
+      verifyTransferFundsMessage('Real')
+      expandDemoWallet()
+      clickAddDerivxButton()
+      verifyDerixCreation('Demo')
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wipWallet/derivx.cy.js:68]: Typo here DeriX instead of DerivX

**Review by meeyun-deriv — APPROVED** [review]
LGTM

File: cypress/e2e/wipWallet/onboarding.cy.js [hunk 2]
```diff
@@ -73,69 +78,23 @@ function onboardingcfdtrading() {
...
-  )
-  cy.findByRole('button', { name: 'Next' }).click()
+  cy.get('[data-test-id="spotlight"]').should('be.visible')
   onboardingcfdtrading()
 }
 
 describe('WALL-3037 & 3039- User Onboarding on Desktop for Fiat Wallets', () => {
   beforeEach(() => {
-    cy.c_login({ app: 'wallets' })
+    cy.c_login('wallets')
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wipWallet/onboarding.cy.js:93]: please revert to old because app needs to be passed in an object

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]
Minor changes requested

**Review by bilalmajid-deriv — APPROVED** [review]

**Pull request merged**